components/ai_chat/browser/constants.h

```diff
@@ -17,6 +17,8 @@ base::span<const webui::LocalizedString> GetLocalizedStrings();
 
 bool UsesLlama2PromptTemplate(const std::string& model);
 
+extern const char kLeoSkuDomain[];
```

Comment:
If this is only used in 1 file (`ai_chat_crendential_manager.cc`) then it can be defined at the top of that file, in an anonymous namespace.